### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by coolboy497** [ukaukaaaa/GazeGNN#4]
When I run the code, I get the following warning, I'm not sure if it will have an effect on the final result, so far it looks like I'm not achieving as much precision as the paper, can you answer my query?
![57FXE9R04V9PV @WAOAKV4C](https://github.com/ukaukaaaa/GazeGNN/assets/65294301/8fdc237e-a01f-4e53-be39-668d53f740f3)

**Comment by ukaukaaaa** [comment]
It seems that your gaze.max() - gaze.min() is 0 here, it means something went wrong in your gaze map. can you check if the npy file you read contain empty gaze? I think this will affect the final result.

**Comment by coolboy497** [comment]
I used the file you provided and checked that the file is not empty, is there any other possible reason for this, can you check?

**Comment by youssefmohana** [comment]
did you solve bug or not ? 

@coolboy497

**Comment by arake001** [comment]
Any solution?

**Comment by arake001** [comment]
I am getting the same issue and the maximum accuracy I get is 74% from the code with the problem. Used the dataset provided. Any solution @coolboy497 @youssefmohana @ukaukaaaa

**Comment by VikasKaja** [comment]
hey I was trying to use your code of Gazegnn for reproducibility and dataset you have uploaded in github and I have never achieved accuracy above 80% my maximum accuracy on test data on the given dataset is 77% beyond that the accuracy never increased and also AUC scores of pneumonia class is very less for me.
Did you use any additional training data to train the model?
which pvig model have given you 83% accuracy on the dataset is it the pvig_ti_224_gelu or the larger ones because larger one always overfits to the data and smaller ones never able to predict pneumonia class.
